examples/save-to-disk/main.go

```diff
@@ -49,32 +86,13 @@ func main() {
 		go func() {
 			ticker := time.NewTicker(time.Second * 3)
 			for range ticker.C {
-				err := peerConnection.SendRTCP(&rtcp.PictureLossIndication{MediaSSRC: track.SSRC()})
-				if err != nil {
-					fmt.Println(err)
+				errSend := peerConnection.SendRTCP(&rtcp.PictureLossIndication{MediaSSRC: track.SSRC()})
+				if errSend != nil {
+					fmt.Println(errSend)
 				}
 			}
 		}()
-
-		if track.Codec().Name == webrtc.VP8 {
-			fmt.Println("Got VP8 track, saving to disk as output.ivf")
-			i, err := ivfwriter.New("output.ivf")
-			if err != nil {
-				panic(err)
-			}
-
-			for {
-				packet, err := track.ReadRTP()
-				if err != nil {
-					panic(err)
-				}
-
-				err = i.AddPacket(packet)
-				if err != nil {
-					panic(err)
-				}
-			}
-		}
+		handleTrack(track)
```

Comment:
I would unfold this a bit, only use a function if it can save some duplication (the indirection makes things a little harder for new devs to read)